### ISSUE TITLE
[Accessibility]: Insufficient color contrast in code monitor create/edit page in dark mode

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#35736]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34195

### Problem description

Two elements in [code monitoring create/edit page](https://sourcegraph.com/code-monitoring/new) have insufficient contrast, in dark mode only:

1. Red error text inside action cards
  ![CleanShot 2022-05-19 at 11 38 03](https://user-images.githubusercontent.com/206864/169375769-046493b7-859c-476e-b3c7-b72e0b9e2e5d.png)
2. Actions heading before any actions have been added
  ![CleanShot 2022-05-19 at 11 38 57](https://user-images.githubusercontent.com/206864/169375916-783f89d7-8d4a-40e2-be65-09046c5d8ad0.png)

### Expected behavior

Colors are updated 

### Additional details

https://github.com/sourcegraph/sourcegraph/issues/35734 also affects this page, but may be a global issue.

Design should suggest appropriate colors and gitstart can update the colors.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by limitedmage** [comment]
@almeidapaulooliveira Can you help choose more appropriate colors here? Thanks!

**Comment by limitedmage** [comment]
Define new semantic variable `--error-text`:
- `--oc-red-8` in light mode
- `--oc-red-6` in dark more
